twifbeddit-app/backend/src/users.js

```diff
@@ -31,58 +31,56 @@ exports.POST = async (_, event) => {
 };
 
 // TODO: Test this. Gets the topics that the user is following.
-exports.get_topics_that_the_user_is_following = async (_, event) => {
-    if (event.queryStringParameters != null)
-        console.error("Parameters currently ignored");
-    const username = await get_user_from_header(event.headers);
-    const topics = await User.findOne({username: username}, 'followed_topics');
-    return {
-        'statusCode': 200,
-        'body': JSON.stringify({
-            topics: topics
-        })
-    }
-};
+exports.GET = async (_, event) => {
+    if (event.queryStringParameters.topicsFollowed != null) {
```

Comment:
What would `topicsFollowed` value even be? It's a weird api. It's also not that useful. A get on users should just return the same thing we return when they log in. It's not like we're concerned about bandwidth

Reply:
We need to get the list of users that a topic follows. I'll just add it to the user model without a query then. Makes sense.